wercker.yml

```diff
@@ -1,74 +1,829 @@
+
+    # some dummy tests for now
+    - script:
+        name: test installed package (py2)
+        code: |
+          echo 'running: python -c "import beprof"'
+          python2 -c 'import beprof'
+          echo 'running: python -c "from beprof.curve import Curve"'
+          python2 -c 'from beprof.curve import Curve'
+          echo 'running: python -c "from beprof.profile import Profile"'
+          python2 -c 'from beprof.profile import Profile'
+          echo "all done"
+
+    - script:
+        name: test installed package (py3)
+        code: |
+          echo 'running: python -c "import beprof"'
+          python3 -c 'import beprof'
+          echo 'running: python -c "from beprof.curve import Curve"'
+          python3 -c 'from beprof.curve import Curve'
+          echo 'running: python -c "from beprof.profile import Profile"'
+          python3 -c 'from beprof.profile import Profile'
+          echo "all done"
+
+
+test_opensuse_tumbleweed_repo:
+  box: opensuse:tumbleweed
+
+  steps:
+    - script:
+        name: prerequisites
+        code: |
+          zypper ref
+          zypper --non-interactive install python python-pip python3-pip
+          python2 -V
+          python3 -V
+          pip2 install --upgrade pip
+          pip2 -V
+          pip3 install --upgrade pip
+          pip3 -V
+
+    - script:
+        name: install numpy from repo
+        code: |
+          zypper --non-interactive in python-numpy python3-numpy
+          echo "---pip2---"
+          pip2 freeze
+          echo "---pip3---"
+          pip3 freeze
+
+    - script:
+        name: package_install
+        code: |
+          pip2 install --no-deps /pipeline/source/beprof-*.whl
+          pip3 install --no-deps /pipeline/source/beprof-*.whl
+
+    # some dummy tests for now
+    - script:
+        name: test installed package (py2)
+        code: |
+          echo 'running: python -c "import beprof"'
+          python2 -c 'import beprof'
+          echo 'running: python -c "from beprof.curve import Curve"'
+          python2 -c 'from beprof.curve import Curve'
+          echo 'running: python -c "from beprof.profile import Profile"'
+          python2 -c 'from beprof.profile import Profile'
+          echo "all done"
+
+    - script:
+        name: test installed package (py3)
+        code: |
+          echo 'running: python -c "import beprof"'
+          python3 -c 'import beprof'
+          echo 'running: python -c "from beprof.curve import Curve"'
+          python3 -c 'from beprof.curve import Curve'
+          echo 'running: python -c "from beprof.profile import Profile"'
+          python3 -c 'from beprof.profile import Profile'
+          echo "all done"
+
+# Archlinux
+
+test_archlinux_repo:
+  box: base/archlinux:latest
+
+  steps:
+    - script:
+        name: prerequisites
+        code: |
+          echo "Server = ftp://ftp.icm.edu.pl/pub/Linux/dist/archlinux/core/os/x86_64/" > /etc/pacman.d/mirrorlist
```

Comment:
Why is that needed ? Add some comment explaining this strange echo.
I am not using official base/archlinux as it looks abandoned.
Instead there is finalduty/archlinux docker. 

See also https://github.com/grzanka/python-pip-versions/blob/master/wercker.yml

I have also my own docker images with preinstalled python. Dockerfiles are here:
https://github.com/grzanka/python-pip-versions/tree/master/Dockerfiles
Images are automatically built here: https://hub.docker.com/r/grzanka/

I am not sure if they can be of some profit or not as wercker installs python really fast.

You can also check files other than README.md here: https://github.com/grzanka/python-pip-versions